### PR TITLE
chore: render children as button content instead of a specific prop

### PR DESCRIPTION
**Pull request by lean8086**
### Description

Replace property `buttonText` with `children` on InquiryMakeOfferButton.
Keep the default value of `Make an Offer`
See [implementation on Force](https://github.com/artsy/force/blob/main/src/v2/Apps/Conversation/Components/ConfirmArtworkButton.tsx#L87).

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Replace property `buttonText` with `children` on InquiryMakeOfferButton - LeandroL

</details>

Need help with something? Have a look on our [docs], or get in touch with us.

[app state migration]: /docs/adding_state_migrations.md
[feature flag]: /docs/developing_a_feature.md
[docs]: /docs/README.md

### CODE REVIEW

File: src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx [hunk 1]
```diff
@@ -143,7 +142,7 @@ export class InquiryMakeOfferButton extends React.Component<InquiryMakeOfferButt
         variant={variant}
         haptic
       >
-        {buttonText ? buttonText : "Make an Offer"}
+        {children || "Make an Offer"}
```
Comment (lean8086) [inline src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx:145]: I'm keeping the same default value from before, although we are using [the other option "Confirm" as default in Force](https://github.com/artsy/force/blob/main/src/v2/Apps/Conversation/Components/ConfirmArtworkButton.tsx#L87).

File: src/app/Scenes/Inbox/Components/Conversations/OpenInquiryModalButton.tsx [hunk 2]
```diff
@@ -67,7 +67,9 @@ export const OpenInquiryModalButton: React.FC<OpenInquiryModalButtonProps> = ({
             conversationID={conversationID}
             onPress={() => trackEvent(tracks.trackTappedMakeOffer(conversationID))}
             replaceModalView={false}
-          />
+          >
+            Make an Offer
```
Comment (lean8086) [inline src/app/Scenes/Inbox/Components/Conversations/OpenInquiryModalButton.tsx:71]: Although this is the default option, I preferred being explicit with the content here (see the entire file to check readability)

File: src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx [hunk 3]
```diff
@@ -12,7 +12,6 @@ export interface InquiryMakeOfferButtonProps {
   // EditionSetID is passed down from the edition selected by the user
```
Comment (araujobarret) [inline src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx:12]: this interface could extend the `ButtonProps` and then we don't need to re-declare some props like variant, disabled, onPress, we would be respecting and drilling atomic elements props in the way we do with other components.

Comment (araujobarret) [reply src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx:12]: was a requested change too from the previous comment in the other PR ☝️

Comment (lean8086) [reply src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx:12]: Woops 👍

File: src/app/Scenes/Inbox/Components/Conversations/OpenInquiryModalButton.tsx (hunk 2 above)
Comment (lean8086) [reply src/app/Scenes/Inbox/Components/Conversations/OpenInquiryModalButton.tsx:71]: nvm, children is not optional on `ButtonProps` so it needs to be explicitly defined.

**Review by araujobarret — APPROVED** [review]
Looks good, thanks for cleaning it up!

**Pull request merged**